controllers/users/usersValidator.js

```diff
@@ -50,5 +50,26 @@ exports.validate = method => {
           .isEmpty()
       ];
     }
+    case UPDATE_USER: {
+      return [
+        body(
+            "firstName",
+            "Please enter a first name between 2 and 30 characters"
```

Comment:
we could also extract messages from 'UPDATE_USER' and 'REGISTER_USER' to messages.js they are repeating themsleves. I forgot about it when i was adding them

Reply:
sure, should I do it in this pull request or create new one with all those messages changes?

Reply:
y you can do it in this pull if you want, if you won't we will create another task for it :D

Reply:
I think I can do it now. Maybe later we'll create pull request which reorganize those messages to separate exports/folders like "validation messages", "server responses" or something like that.

Reply:
@kamilmoskal I support you in that. We should extract all messages into messages file